packages/block/src/block.ts

```diff
@@ -261,24 +262,6 @@ export class Block {
     return stringError ? errors : errors.length === 0
   }
 
-  /**
-   * Performs the following consistency checks on the block:
-   *
-   * - Value checks on the header fields
-   * - Signature and gasLimit validation for included txs
-   * - Validation of the tx trie
-   * - Consistency checks and header validation of included uncles
-   *
-   * Throws if invalid.
-   *
-   * @param blockchain - validate against an @ethereumjs/blockchain
-   * @param onlyHeader - if should only validate the header (skips validating txTrie and unclesHash) (default: false)
-   */
-  async validate(blockchain: Blockchain, onlyHeader: boolean = false): Promise<void> {
-    await this.header.validate(blockchain)
-    await this.validateUncles(blockchain)
-    await this.validateData(onlyHeader)
```

Comment:
Again, can't comment on the method itself: I would have a tendency to at least rename this to `validateBlockData()`, I always stumble upon this name since data is so ambiguous. If you go along maybe just do.

Reply:
Done